packages/opentelemetry-tracing/test/export/TestStackContextManager.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ContextManager, Context } from '@opentelemetry/context-base';
+
+/**
+ * A test-only ContextManager that uses an in-memory stack to keep track of
```

Comment:
Why not use the existing stack context manager?

Reply:
That would require adding a dependency to opentelemetry-web only for the purpose of running this test. That seemed even more awkward than this. Or perhaps there's some reorganization that can/should happen?

Reply:
Hmm, maybe a follow up is to move it into core as there is nothing inherently web-specific about it. For now, it's fine to keep this here I guess.

Reply:
you can add this dependency in dev dependency, if there will be any changes to context manager the web will never fail again

Reply:
moving into core would potentially make sense to me. at the time, all the context managers seemed to be in their usage-specific modules so i hesitated to attempt things like that being new, prior to feedback. up for doing that with this PR or separately. i suppose a separate PR might be better for limiting scope of changes/review?

Reply:
I think this can be handled in follow up